src/sidebars/sidebar-controller.js

```diff
@@ -28,10 +28,20 @@
         }
 
         searchClick() {
+            if (this._showSearches) {
+                return;
+            }
```

Comment:
I'm concerned by this code, especially given #707. Adding different functionality for the use of behaviour tracking doesn't feel right.